backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -33,6 +38,7 @@
 public class UsersController {
 
 	private final UserModelMapper userModelMapper = Mappers.getMapper(UserModelMapper.class);
+	private final UserAttributeMapper userAttributeMapper = Mappers.getMapper(UserAttributeMapper.class);
```

Comment:
`UserAttributeMapper` is being referenced but it is not being used. Can we remove this line?

Reply:
removed